nss-cohort-40/bangazon-ecommerce-api-lumberjacks-back
# Auth

# Description
Back end authentication

 Fixes # (issue)
 ## Type of change
 Please delete options that are not relevant.

 # Testing Instructions
Test in conjunction with front-end auth to ensure you can register and login a new user.

 # Checklist:
